src/components/Display.jsx

```diff
@@ -1,21 +1,18 @@
 /* eslint-disable react/prefer-stateless-function */
-import React, { Component } from 'react';
+import React from 'react';
 import PropTypes from 'prop-types';
 
-export default class Display extends Component {
-  render() {
-    const { display } = this.props;
-    return (
-      <div
-        className="display span-4"
-        id="display"
-        type="text"
-        disabled
-      >
-        {display || '0'}
-      </div>
-    );
-  }
+export default function Display({ display }) {
```

Comment:
- [ ] [OPTIONAL] Please use arrow functions since you should use ES6 syntax , I strongly recommend you to take them into account as they can make your code better.